README.md

```diff
@@ -45,6 +45,14 @@ need to re-build those files using:
 npm run build
 ```
 
+## Start Local Server
+
+To view the site locally, just run:
+
+```
+npm run start
```

Comment:
You can also just do `npm start`

Reply:
do you want me to change it to that? I just wanted it clear on the readme

Reply:
It's a nit, don't care either way, i Iike npm start, it's faster.